docs/concepts/content-types.md

```diff
@@ -32,71 +32,6 @@ Once adopted, a standard content type is bundled in XMTP client SDKs. A develope
 
 Here are the current standard content types:
 
-### Text content type
```

Comment:
@fabriguespe I think we keep this "Text content type" section? It is still a standard content type. We just need to remove the Composite content type section starting at line 85? =)

Reply:
You are right let's keep it!